frontend/packages/core/src/AppLayout/notifications.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import styled from "@emotion/styled";
+import {
+  Box,
+  ClickAwayListener,
+  Grow,
+  IconButton,
+  MenuList,
+  Paper,
+  Popper,
+} from "@material-ui/core";
+import NotificationsIcon from "@material-ui/icons/Notifications";
+
+const StyledNotificationsIcon = styled(IconButton)`
+  color: #ffffff;
+  margin-right: 1rem;
+  border-radius: 8px;
+  padding: 8px;
+  &:hover {
+    background: #2d3db4;
+  }
+  &:active {
+    background: #2938a5;
+  }
+`;
+
+const Notifications: React.FC = () => {
+  const [open, setOpen] = React.useState(false);
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
+
+  function handleListKeyDown(event) {
+    if (event.key === "Tab") {
+      event.preventDefault();
+      setOpen(false);
+    }
+  }
+
+  return (
+    <Box>
+      <StyledNotificationsIcon
+        ref={anchorRef}
+        edge="end"
+        aria-controls={open ? "notifcation-options" : undefined}
+        aria-haspopup="true"
+        onClick={handleToggle}
+      >
+        <NotificationsIcon />
+      </StyledNotificationsIcon>
+      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{ transformOrigin: placement === "bottom" ? "center top" : "center bottom" }}
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <MenuList
+                  autoFocusItem={open}
+                  id="notifcation-options"
+                  onKeyDown={handleListKeyDown}
+                />
+              </ClickAwayListener>
+            </Paper>
+          </Grow>
+        )}
+      </Popper>
+    </Box>
+  );
+};
+
+export default Notifications;
```

Comment:
It might be useful to have a notifications story so we can test the whole component (e.g. the popper and such) in isolation from the header.

Reply:
I pushed up a notifications story. At the moment, we won't see the popper b/c we don't have menu items.

Reply:
Awesome! I was thinking that we could have an interface for the component where it takes in the notifications that we want to render. I know we don't have designs for this yet so feel free to punt but ideally we pass in notifications that would allow us to test the component. Not blocking at this point but something to keep in mind.

Reply:
okay got it! will leave a to-do, I'll follow up to add an interface. I think the metadata table interface would be a good one to pattern